UefiPayloadPkg/PayloadLoaderPeim/FitPayloadLoaderPeim.c

```diff
@@ -144,13 +181,14 @@ PeiLoadFileLoadPayload (
   *ImageSizeArg    = Context.PayloadSize;
   *EntryPoint      = Context.PayloadEntryPoint;
 
+ #if (FixedPcdGetBool (PcdHandOffFdtEnable))
+
   Status = PeiServicesInstallPpi (&mEndOfPeiSignalPpi);

```

Comment:
Good catch!
In old implementation EndOfPeiSignalPpi will be installed twice when PcdHandOffFdtEnable == 0:
1. Here
2. HandOffToDxeCore() later

Fix looks good to me.